deps/build.jl

```diff
@@ -25,6 +25,7 @@ provides(BuildProcess, Autotools(libtarget = "src/.libs/libzmq." * dlext), zmq)
 provides(AptGet, "libzmq3", zmq, os = :Linux)
 provides(Yum, "czmq", zmq, os = :Linux)
 provides(Pacman, "zeromq", zmq, os = :Linux)
+provides(BSDPkg, "czmq", zmq, os=:FreeBSD)
```

Comment:
this requires a minimum bindeps version of 0.6.0 https://github.com/JuliaLang/BinDeps.jl/commit/264e086062f3c39adba7997f187bd54bc0e6b218